provider/terraform_object_library.rb

```diff
@@ -53,6 +53,16 @@ def copy_common_files(output_folder, _version_name)
                         'third_party/validator/compute_instance.go'],
                        ['google/sql_database_instance.go',
                         'third_party/validator/sql_database_instance.go'],
+                       ['google/iam_helpers.go',
```

Comment:
At some point you'll probably want to support a splat syntax for `third_party/validator/*` files. No action needed now though.

Reply:
Thanks for heads up!